internal/batches/executor/run_steps.go

```diff
@@ -303,6 +303,11 @@ func executeSingleStep(
 		args = append(args, "--mount", fmt.Sprintf("type=bind,source=%s,target=%s,ro", source.Name(), target))
 	}
 
+	// mount any paths on the local system to the docker container
+	for _, mount := range step.Mount {
+		args = append(args, "--mount", fmt.Sprintf("type=bind,source=%s,target=%s,ro", mount.Path, mount.Mountpoint))
```

Comment:
Edge case: what happens if `mount.Path` or `mount.Mountpoint` include a literal comma?

Reply:
That sounds scary 😬 I think the same question applies to the `files` property.

Reply:
If a comma is provided, Docker gets angry because it thinks the value after the comma is a field and it is not in the right format.

```text
invalid argument "type=bind,source=/Users/randell/Documents/development/batch-spec-playground/some,bad/foo.py,target=/tmp/some/foo.py,ro" for "--mount" flag: invalid field 'bad/foo.py' must be a key=value pair

invalid argument "type=bind,source=/Users/randell/Documents/development/batch-spec-playground/foo.py,target=/tmp/some,bad/foo.py,ro" for "--mount" flag: invalid field 'bad/foo.py' must be a key=value pair
```

Regardless, I'll add a check for this character and any others.